pkg/agent/agent.go

```diff
@@ -215,6 +225,11 @@ func (i *Initializer) Initialize() error {
 		return err
 	}
 
+	if err := i.routeClient.Initialize(i.nodeConfig); err != nil {
+		return err
+	}
+
+	klog.Infof("Agent initialized NodeConfig=%v\nencapMode=%s", i.nodeConfig, config.TrafficEncapMode)
```

Comment:
nit: better not to introduce newline which could break log format.

Reply:
done